themes/src/main/resources/theme/base/email/messages/messages_en.properties

```
@@ -39,18 +39,11 @@ requiredAction.VERIFY_EMAIL=Verify Email
 requiredAction.CONFIGURE_RECOVERY_AUTHN_CODES=Generate Recovery Codes
 
 # units for link expiration timeout formatting
-linkExpirationFormatter.timePeriodUnit.seconds=seconds
-linkExpirationFormatter.timePeriodUnit.seconds.1=second
-linkExpirationFormatter.timePeriodUnit.minutes=minutes
-linkExpirationFormatter.timePeriodUnit.minutes.1=minute
-#for language which have more unit plural forms depending on the value (eg. Czech and other Slavic langs) you can override unit text for some other values like this: 
-#linkExpirationFormatter.timePeriodUnit.minutes.2=minuty
-#linkExpirationFormatter.timePeriodUnit.minutes.3=minuty
-#linkExpirationFormatter.timePeriodUnit.minutes.4=minuty
-linkExpirationFormatter.timePeriodUnit.hours=hours
-linkExpirationFormatter.timePeriodUnit.hours.1=hour
-linkExpirationFormatter.timePeriodUnit.days=days
-linkExpirationFormatter.timePeriodUnit.days.1=day
+linkExpirationFormatter.timePeriodUnit.seconds={0,choice,0#seconds|1#second|1<seconds}
```

ChoiceFormat is news to me ;)

Can't this be simplified to:
```suggestion
linkExpirationFormatter.timePeriodUnit.seconds=0#seconds|1#second|1<seconds
```

---

At least that's the format I get from trying to understand https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/text/ChoiceFormat.html

---

IMHO the translation tools would recognize a Java message format `{0,choice,...}` and then validate it accordingly. So it is the best format for translation.

For some of the languages which didn't have any plurals (like `ja` and `tr` locales), where this is then very simple, although this is the exception.

So I'd like to proceed with this setup.

---

Looking at https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/text/MessageFormat.html I'm fine with that as I now understand the syntax